lib/ansible/plugins/test/core.py

```diff
@@ -78,14 +79,52 @@ def finished(result):
         raise errors.AnsibleFilterError("The 'finished' test expects a dictionary")
     if 'finished' in result:
         # For async tasks return status
-        # NOTE: The value of finished it 0 or 1, not False or True :-/
+        # NOTE: The value of finished is 0 or 1, not False or True :-/
         return result.get('finished', 0) == 1
     else:
         # For non-async tasks warn user, but return as finished
         display.warning("The 'finished' test expects an async task, but a non-async task was tested")
         return True
 
 
+def test_boolean(value):
+    ''' Return true if the object is a boolean value '''
+    return value is True or value is False
+
+
+def test_false(value):
+    ''' Return true if the object is False '''
+    return value is False
+
+
+def test_true(value):
+    ''' Return true if the object is True '''
+    return value is True
+
+
+# NOTE: The existing Jinja2 'number' test also matches booleans and floats
+def test_integer(value):
+    ''' Return true if the object is an integer '''
+    return isinstance(value, integer_types) and value is not True and value is not False
+
+
+# NOTE: The existing Jinja2 'number' test also matches booleans and integers
+def test_float(value):
+    ''' Return true if the object is a float '''
+    return isinstance(value, float)
+
+
+# NOTE: The existing Jinja2 'sequence' test also matches strings and dictionaries
+def test_list(value):
```

Comment:
This code is already available in `ansible.module_utils.common.collections.is_sequence` for broader use.
So please follow earlier suggestion to reuse it: https://github.com/ansible/ansible/pull/37517#discussion_r192379339